grimoirelab/toolkit/datetime.py

```diff
@@ -83,7 +84,14 @@ def datetime_to_utc(ts):
     if not ts.tzinfo:
         ts = ts.replace(tzinfo=dateutil.tz.tzutc())
 
-    return ts.astimezone(dateutil.tz.tzutc())
+    try:
+        ts = ts.astimezone(dateutil.tz.tzutc())
+    except ValueError:
+        logger.warning("Date %s str does not have a valid timezone", ts)
+        logger.warning("Date converted removing timezone info")
```

Comment:
I would changed it by `Date converted to UTC removing timezone info` (easier to trace in logs).